.ci_support/win_c_compilervs2015cxx_compilervs2015python3.6vc14.yaml

```diff
@@ -11,25 +11,25 @@ cxx_compiler:
 expat:
 - '2.2'
 gdal:
-- '2.2'
+- 2.2.4
 geos:
 - 3.6.2
 hdf4:
 - '4.2'
 hdf5:
-- 1.10.3
+- 1.10.2
 icu:
 - '58'
 jpeg:
 - '9'
 kealib:
-- 1.4.9
+- 1.4.10
 libkml:
 - '1.3'
 libnetcdf:
-- '4.6'
+- 4.4.1.1
```

Comment:
This is not good :-/ only `vc9` should be changed.

Reply:
If I'm understanding you correctly, this is setting the vc9-only libnetcdf version on a python 3.6 recipe, right?

Reply:
This config file is `win_c_compilervs2015cxx_compilervs2015python3.6vc14.yaml`, which means it is `win and vc14`, so when I set the pre-selector for the new pins to be `win and vc<14` this one should not be modified. I expected changes only in the `win_c_compilervs2008cxx_compilervs2008python2.7vc9.yaml` config file.